### DISCUSSION TITLE
Deployment error: Resource is limited (more than 100, code: "api-deployments-free-per-day") - I'm nowhere near that many deploys

### DISCUSSION THREAD

**Discussion by alexanderbnelson** [vercel/community#4350] [Help]
### Summary

I'm nowhere near 100 daily deployments as per Vercel Limits on hobby plans [](https://vercel.com/docs/limits/overview)

I've made 9 commits in the last 24 hours and only 4 of those have deployed. Vercel stopped deploying 17 hours ago for me and started giving me this Resource is limited error. I've tried forcing a deploy with a webhook and tried deploying with Vercel CLI. Nothing works. What's going on here?

### Steps to Reproduce

N/A

**Comment by amyegan** [comment]
Do you have a deploy hook set up for a project that might be generating new deployments for something like CMS content updates? Depending on how it was configured, that could cause the [number of builds](https://vercel.com/docs/limits/usage#number-of-builds) to get high very quickly.

**Comment by qn2060** [comment]
Meeting the same thing...  I just deployed 4 times in the past 24 hours.

**Comment by JayeshVP24** [comment]
i'm also facing the same error, have 4 deployments in last 24 hours and yet unable to get new commits deployed, when manually adding commit, it throws an error showing i've crossed the 100 build limit

![image](https://github.com/vercel/community/assets/59532173/4c2f2ea9-13e6-4da2-be4e-041bef592819)

**Comment by phillipmalboeuf** [comment]
I'm also getting this "more than 100" error without having reached that number at all.

**Comment by VenyaBrodetskiy** [comment]
I have same problem. Number of builds today is just 3, but I got error from vercel bot, deployment is not started.
![image](https://github.com/vercel/community/assets/106010706/aff271e6-a926-43b0-80fc-af5c9d75a91b)
![image_2024-04-19_12-29-57](https://github.com/vercel/community/assets/106010706/cb0d148a-551b-4293-a5f3-8f1a1b47559f)

**Reply by amyegan** [reply]
Hey @VenyaBrodetskiy. Are your deployments working now, or is this still a problem for you?

**Reply by VenyaBrodetskiy** [reply]
I had 3 hours ban as you can see on the screenshot. Once it finished, new deployment worked fine
